libs/ngrx-toolkit/src/lib/with-mutations.spec.ts

```diff
@@ -0,0 +1,537 @@
+import { fakeAsync, TestBed, tick } from '@angular/core/testing';
```

Comment:
TODO: Generally speaking, once we make `rxMutation` standalone, most tests of `rxMutation` will cover already what we have here.